sdk/sqlmanagement/Microsoft.Azure.Management.Sql/tests/LedgerDigestUploadsTests.cs

```diff
@@ -0,0 +1,90 @@
+
+                // Verify the initial GET request contains the default settings (disabled)
+                Assert.Equal(LedgerDigestUploadsState.Disabled, defaultResponse.State);
+
+                // Create new configuration with ACL endpoint
+                LedgerDigestUploads aclUploadConfiguration = new LedgerDigestUploads
+                {
+                    DigestStorageEndpoint = aclEndpoint,
+                };
+
+                // Set ledger digest upload configuration for database
+                sqlClient.LedgerDigestUploads.CreateOrUpdate(resourceGroup.Name, server.Name, database.Name, aclUploadConfiguration);
+
+                // Get the updated ledger digest upload properties
+                LedgerDigestUploads aclResponse = sqlClient.LedgerDigestUploads.Get(resourceGroup.Name, server.Name, database.Name);
+
+                // Verify that the GET request contains the updated settings
+                Assert.Equal(LedgerDigestUploadsState.Enabled, aclResponse.State);
+                Assert.Equal(aclEndpoint, aclResponse.DigestStorageEndpoint);
+
+                // Disable digest uploading on database
+                sqlClient.LedgerDigestUploads.Disable(resourceGroup.Name, server.Name, database.Name);
+
+                // Get the updated ledger digest upload properties
+                LedgerDigestUploads disabledResponse = sqlClient.LedgerDigestUploads.Get(resourceGroup.Name, server.Name, database.Name);
+
+                // Verify that the GET request contains the disabled settings
+                Assert.Equal(LedgerDigestUploadsState.Disabled, disabledResponse.State);
+            }
+        }
+
+        [Fact]
+        public void TestCreateLedgerDatabase()
+        {
+            using (SqlManagementTestContext context = new SqlManagementTestContext(this))
+            {
+                SqlManagementClient sqlClient = context.GetClient<SqlManagementClient>();
+
+                ResourceGroup resourceGroup = context.CreateResourceGroup();
+                Server server = context.CreateServer(resourceGroup);
+
+                // Create database with IsLedgerOn 
+                string databaseName = SqlManagementTestUtilities.GenerateName();
+                Database ledgerDb = sqlClient.Databases.CreateOrUpdate(
+                    resourceGroup.Name,
+                    server.Name,
+                    databaseName,
+                    new Database()
+                    {
+                        Location = server.Location,
+                        IsLedgerOn = true
+                    });
+
+                // Get the created database and validate IsLedgerOn parameter is set
+                Database databaseResponse = sqlClient.Databases.Get(resourceGroup.Name, server.Name, databaseName);
+                Assert.True(databaseResponse.IsLedgerOn);
+            }
+        }
```

Comment:
The test failure you are seeing is caused by a mismatch between the class name for your test path and the path segment in the recording files: LedgerDigestUploadsTests (path segment) -> LedgerDigestUploadTests (actual class name)